mappings/net/minecraft/client/render/entity/LivingEntityRenderer.mapping

```diff
@@ -2,16 +2,37 @@ CLASS net/minecraft/class_922 net/minecraft/client/render/entity/LivingEntityRen
 	FIELD field_21011 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_4737 model Lnet/minecraft/class_583;
 	FIELD field_4738 features Ljava/util/List;
+	METHOD <init> (Lnet/minecraft/class_898;Lnet/minecraft/class_583;F)V
+		ARG 1 dispatcher
+		ARG 2 model
+		ARG 3 shadowRadius
 	METHOD method_18656 getYaw (Lnet/minecraft/class_2350;)F
+		ARG 0 direction
+	METHOD method_23185 getAnimationCounter (Lnet/minecraft/class_1309;F)F
+		ARG 1 entity
+		ARG 2 tickDelta
+	METHOD method_23622 getOverlay (Lnet/minecraft/class_1309;F)I
+		ARG 0 entity
+		ARG 1 animationCounter
 	METHOD method_4039 getLyingAngle (Lnet/minecraft/class_1309;)F
+		ARG 1 entity
 	METHOD method_4042 scale (Lnet/minecraft/class_1309;Lnet/minecraft/class_4587;F)V
 		ARG 1 entity
+		ARG 2 matrix
 	METHOD method_4044 getHandSwingProgress (Lnet/minecraft/class_1309;F)F
 		ARG 1 entity
 		ARG 2 tickDelta
 	METHOD method_4045 getAge (Lnet/minecraft/class_1309;F)F
 		ARG 1 entity
 		ARG 2 tickDelta
 	METHOD method_4046 addFeature (Lnet/minecraft/class_3887;)Z
+		ARG 1 feature
+	METHOD method_4056 isVisible (Lnet/minecraft/class_1309;Z)Z
+		ARG 1 entity
+		ARG 2 xrayEnabled
```

Comment:
What does this do? When would xray be enabled?

Reply:
The meaning is pretty clear (passing true here cauzes the method to always return true).

I didn't check, but I suspect it's used for that feature where entity outlines are rendered through blocks. I don't remember what it's called.